assets/css/app.css

```
@@ -1,179 +1,172 @@
-   
-   Architecture Documentation: docs/CSS_ARCHITECTURE_SPECIFICATION.md
-   ============================================================================ */
+/* =============================================================================
+  GOOGLE FONTS & THEME-SPECIFIC TYPOGRAPHY
+  ============================================================================= */
 
-/* ============================================================================
-   BASE THEME FOUNDATION
-   ============================================================================ */
+/* Import Google Fonts for Themes */
+@import url('https://fonts.googleapis.com/css2?family=Pacifico&family=Creepster&family=Bangers&family=Press+Start+2P&family=Caveat:wght@400;700&family=Righteous&family=Bungee&family=Fascinate&family=Orbitron:wght@400;700;900&family=Bebas+Neue&family=Permanent+Marker&family=Indie+Flower&family=Dancing+Script:wght@400;700&family=Great+Vibes&family=Zilla+Slab:wght@400;700&family=Fredoka:wght@400;700&family=Lobster&family=Amatic+SC:wght@400;700&family=Comic+Neue:wght@400;700&family=Kalam:wght@400;700&family=Patrick+Hand&family=Homemade+Apple&family=Rock+Salt&family=Reenie+Beanie&family=Just+Another+Hand&display=swap');
```

Biome (1.9.4)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)